tanzu-build-service/dependencies.hbs.md

```diff
@@ -164,22 +164,22 @@ The `full` dependencies contain the following buildpacks in Tanzu Application Pl
 
 | Buildpack | Version | Supported Stacks |
 |-----------|---------|------------------|
-| Java Buildpack for VMware Tanzu | 8.0.0 | Bionic, Jammy |
-| Java Native Image Buildpack for Tanzu | 6.39.0 | Bionic, Jammy |
-| .NET Core Buildpack for VMware Tanzu | 1.19.2 | Bionic, Jammy |
-| Node.js Buildpack for VMware Tanzu | 1.17.1 | Bionic, Jammy |
-| Python Buildpack for VMware Tanzu | 2.3.1 | Bionic, Jammy |
-| Ruby Buildpack for VMware Tanzu | 1.5.0 | Bionic, Jammy |
-| Go Buildpack for VMware Tanzu | 2.0.4 | Bionic, Jammy |
+| Java Buildpack for VMware Tanzu | [8.0.0](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-release-notes-tanzu-java-release-notes.html#tanzu-java-buildpack-8.0.0) | Bionic, Jammy |
+| Java Native Image Buildpack for Tanzu | [6.39.0](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-release-notes-tanzu-java-native-image-release-notes.html#tanzu-java-native-image-buildpack-6.39.0) | Bionic, Jammy |
+| .NET Core Buildpack for VMware Tanzu | [1.19.2](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-dotnet-core-release-notes.html#tanzu.net-core-buildpack-1.19.2-full) | Bionic, Jammy |
+| Node.js Buildpack for VMware Tanzu | [1.17.1](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-nodejs-release-notes.html#tanzu-node.js-buildpack-1.17.0-full) | Bionic, Jammy |
+| Python Buildpack for VMware Tanzu | [2.3.1](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-python-release-notes.html#tanzu-python-buildpack-2.3.1-full) | Bionic, Jammy |
+| Ruby Buildpack for VMware Tanzu | [1.5.0](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-ruby-release-notes.html#tanzu-ruby-buildpack-1.5.0-full) | Bionic, Jammy |
+| Go Buildpack for VMware Tanzu | [2.0.4](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-go-release-notes.html#tanzu-go-buildpack-2.0.4-full) | Bionic, Jammy |
 | PHP Buildpack for VMware Tanzu | 2.0.0 | Bionic, Jammy |
```

Comment:
I can't find the release note entry for this?

Reply:
I think PHP doesn't have one yet because it's so new.